proto/em/authority/v1beta1/query.proto

```diff
@@ -0,0 +1,27 @@
+syntax = "proto3";
+package em.authority.v1beta1;
+
+import "gogoproto/gogo.proto";
+import "cosmos/base/v1beta1/coin.proto";
+import "google/api/annotations.proto";
+
+option go_package = "github.com/e-money/em-ledger/x/authority/types";
+
+service Query {
+  rpc GasPrices(QueryGasPricesRequest) returns (QueryGasPricesResponse) {
+    // todo (reviewer) : this and others doe the rest route make sense
+    option (google.api.http).get = "/e-money/authority/v1beta1/gasprices";
```

Comment:
the `v?beta` element in the URL paths, do not add value and undermine confidence in the public usage of the APIs.